app/javascript/components/shared_components/Logo.jsx

```diff
@@ -1,15 +1,24 @@
 import React from 'react';
 import Image from 'react-bootstrap/Image';
+import PropTypes from 'prop-types';
 import useSiteSetting from '../../hooks/queries/site_settings/useSiteSetting';
 
-export default function Logo() {
+export default function Logo({ className }) {
```

Comment:
Can we pass it a size param instead of replacing the entire className?

Reply:
We could use a wrapper instead to provide the CSS classes. And pass the width and height as props. Planning to use the same component for the navbar logo as well.